tests/integration_tests/modules/test_package_update_upgrade_install.py

```diff
@@ -0,0 +1,76 @@
+"""Integration test for the package update upgrade install module.
+
+This test module asserts that packages are upgraded/updated during boot
+with the ``package_update_upgrade_install`` module. We are also testing
+if we can install new packages during boot too.
+
+(This is ported from
+``tests/cloud_tests/testcases/modules/package_update_upgrade_install.yaml``.)
+
+NOTE: the testcase for this looks for the command in history.log as
+      /usr/bin/apt-get..., which is not how it always appears. it should
+      instead look for just apt-get...
+NOTE: this testcase should not require 'apt_up_out', and should look for a
+      call to 'apt-get upgrade' or 'apt-get dist-upgrade' in cloud-init.log
+      rather than 'Calculating upgrade...' in output."""
```

Comment:
Looking at https://github.com/canonical/cloud-init/blob/master/tests/cloud_tests/releases.yaml, `trusty` is the only case where `apt_up_out` is false, so I think we can drop this note:

```suggestion
```